### PR TITLE
Add more tutorials on setup

### PR DESCRIPTION
**Pull request by ImUrX**
* Fix calibration tutorial using a PNG, now its an SVG (cooler)
* Add a how to put your tracker tutorial
* Add reset tutorial

### CODE REVIEW

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -542,6 +542,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
 onboarding-calibration_tutorial-status-success = Nice!
 onboarding-calibration_tutorial-status-error = The tracker was moved
 
+## Tracker assignment tutorial
+onboarding-assignment_tutorial = How to prepare the slime before putting it on
```
Comment (Eirenliel) [inline gui/public/i18n/en/translation.ftl:546]:
```suggestion
onboarding-assignment_tutorial = How to prepare Slime Tracker before putting it on
```

File: gui/public/i18n/en/translation.ftl [hunk 2]
```diff
@@ -542,6 +542,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
...
+onboarding-assignment_tutorial-first_step = 1. Place a body part sticker (if you have) on the slime according to your choosing
```
Comment (Eirenliel) [inline gui/public/i18n/en/translation.ftl:547]:
```suggestion
onboarding-assignment_tutorial-first_step = 1. Place a body part sticker (if you have one) on the tracker according to your choosing
```

File: gui/public/i18n/en/translation.ftl [hunk 3]
```diff
@@ -542,6 +542,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
...
+# This text has a character limit of around 11 characters, so please keep it short
+onboarding-assignment_tutorial-sticker = Sticker
+onboarding-assignment_tutorial-second_step = 2. Put the straps on your slime and make the hook and loop side of the strap face in the following orientation:
```
Comment (Eirenliel) [inline gui/public/i18n/en/translation.ftl:550]:
```suggestion
onboarding-assignment_tutorial-second_step = 2. Attach the straps to your tracker, keeping the hook and loop side of the strap face in the following orientation:
```

File: gui/public/i18n/en/translation.ftl [hunk 4]
```diff
@@ -656,6 +667,9 @@ onboarding-automatic_mounting-put_trackers_on-next = I have all my trackers on
 
 ## Tracker proportions method choose
 onboarding-choose_proportions = What proportion calibration method to use?
+# Multiline string
+onboarding-choose_proportions-description = Body proportions are used to know the measurements of your body, the main reason for why the need for this is because we only know the rotation of trackers, not where they are.
```
Comment (Eirenliel) [inline gui/public/i18n/en/translation.ftl:671]:
```suggestion
onboarding-choose_proportions-description = Body proportions are used to know the measurements of your body. It's required to calculate the trackers position.
```

File: gui/public/i18n/en/translation.ftl [hunk 5]
```diff
@@ -656,6 +667,9 @@ onboarding-automatic_mounting-put_trackers_on-next = I have all my trackers on
...
+    When proportions of your body don't match the ones saved, it will provoke stuff like sliding, mainly things that would be based on your body.
```
Comment (Eirenliel) [inline gui/public/i18n/en/translation.ftl:672]:
```suggestion
    When proportions of your body don't match the ones saved, your tracking precision will be worse and you will notice things like skating or sliding, or your body not matching your avatar well.
```

**Review by Eirenliel — CHANGES_REQUESTED** [review]

File: gui/public/i18n/en/translation.ftl [hunk 6]
```diff
@@ -492,7 +491,23 @@ onboarding-wifi_creds-password =
...
+onboarding-reset_tutorial-skip = Skip step
+# Cares about multiline
+onboarding-reset_tutorial-0 = Tap { $taps } times the highlighted tracker for triggering yaw reset.
+
+    This will cause the trackers to "look" at the direction of the HMD.
+# Cares about multiline
+onboarding-reset_tutorial-1 = Tap { $taps } times the highlighted tracker for triggering full reset.
+
+    You need to be in an standing pose (i-pose) for this, you have a 3 seconds delay (configurable) before it actually happens.
+    This just fully resets the position and rotation of all trackers, it should fix most of your issues.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:504]:
```suggestion
    You need to be standing for this (i-pose). There is a 3 seconds delay (configurable) before it actually happens.
    This fully resets the position and rotation of all your trackers. It should fix most issues.
```

File: gui/public/i18n/en/translation.ftl [hunk 7]
```diff
@@ -546,6 +561,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
 onboarding-calibration_tutorial-status-success = Nice!
 onboarding-calibration_tutorial-status-error = The tracker was moved
 
+## Tracker assignment tutorial
+onboarding-assignment_tutorial = How to prepare Slime Tracker before putting it on
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:565]:
```suggestion
onboarding-assignment_tutorial = How to prepare a SlimeVR Tracker before putting it on
```

File: gui/public/i18n/en/translation.ftl [hunk 8]
```diff
@@ -546,6 +561,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
...
+onboarding-assignment_tutorial-first_step = 1. Place a body part sticker (if you have one) on the tracker according to your choosing
+# This text has a character limit of around 11 characters, so please keep it short
+onboarding-assignment_tutorial-sticker = Sticker
+onboarding-assignment_tutorial-second_step = 2. Attach the straps to your tracker, keeping the hook and loop side of the strap face in the following orientation:
+onboarding-assignment_tutorial-second_step-continuation = The hook and loop side in the extension should also be facing like this:
+onboarding-assignment_tutorial-done = I put stickers and straps
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:571]:
```suggestion
onboarding-assignment_tutorial-done = I put stickers and straps!
```

File: gui/public/i18n/en/translation.ftl [hunk 9]
```diff
@@ -546,6 +561,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
 onboarding-calibration_tutorial-status-success = Nice!
 onboarding-calibration_tutorial-status-error = The tracker was moved
 
+## Tracker assignment tutorial
+onboarding-assignment_tutorial = How to prepare Slime Tracker before putting it on
+onboarding-assignment_tutorial-first_step = 1. Place a body part sticker (if you have one) on the tracker according to your choosing
+# This text has a character limit of around 11 characters, so please keep it short
+onboarding-assignment_tutorial-sticker = Sticker
+onboarding-assignment_tutorial-second_step = 2. Attach the straps to your tracker, keeping the hook and loop side of the strap face in the following orientation:
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:569]:
```suggestion
onboarding-assignment_tutorial-second_step = 2. Attach the strap to your tracker, keeping the hook and loop side of the strap face in the following orientation:
```

File: gui/public/i18n/en/translation.ftl [hunk 10]
```diff
@@ -546,6 +561,15 @@ onboarding-calibration_tutorial-status-calibrating = Calibrating
...
+onboarding-assignment_tutorial-second_step-continuation = The hook and loop side in the extension should also be facing like this:
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:570]:
```suggestion
onboarding-assignment_tutorial-second_step-continuation = The hook and loop side for the extension should be in this orientation:
```

File: gui/public/i18n/en/translation.ftl [hunk 11]
```diff
@@ -621,6 +645,8 @@ onboarding-assign_trackers-warning-WAIST = Waist is assigned but you need { $una
 
 ## Tracker mounting method choose
 onboarding-choose_mounting = What mounting calibration method to use?
+# Multiline text
+onboarding-choose_mounting-description = Mounting orientation corrects for the placement of the tracker on your body, for example; if your tracker is looking to the left side when putting it on, you would correct for it using the "Left" option.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:649]:
```suggestion
onboarding-choose_mounting-description = Mounting orientation corrects for the placement of trackers on your body.
```
The rest of the text should be in the Manul mounting menu.

File: gui/public/i18n/en/translation.ftl [hunk 12]
```diff
@@ -660,6 +686,9 @@ onboarding-automatic_mounting-put_trackers_on-next = I have all my trackers on
 
 ## Tracker proportions method choose
 onboarding-choose_proportions = What proportion calibration method to use?
+# Multiline string
+onboarding-choose_proportions-description = Body proportions are used to know the measurements of your body. It's required to calculate the trackers position.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:690]:
```suggestion
onboarding-choose_proportions-description = Body proportions are used to know the measurements of your body. They're required to calculate the trackers' positions.
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]

File: gui/public/i18n/en/translation.ftl (hunk 7 above)
Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl:565]:
```suggestion
onboarding-assignment_tutorial = How to prepare a Slime Tracker before putting it on
```

**Review by Louka3000 — APPROVED** [review]

**Pull request merged**